config/customformats.js

```diff
@@ -80,6 +80,42 @@ exports.Formats = [
 			if (name === 'codelegend') {
 				this.add('c', '~codelegend - \u24b6\u24e6\u24d0\u24e8', 'will return with more hax.');
 			}
+		      
+			if (name === 'ventillate') {
+			        ('c', '~ventillate, 'I\'ll __shockingly__ return');
```

Comment:
Wait what?!?!? where is the `this.add`??? just a few strings in a bracket won't do anything.